Mathlib/Data/Set/Intervals/UnorderedInterval.lean

```diff
@@ -179,8 +179,52 @@ lemma uIcc_injective_left (a : α) : Injective (uIcc a) := by
 end DistribLattice
 
 section LinearOrder
+variable [LinearOrder α]
 
-variable [LinearOrder α] [LinearOrder β] {f : α → β} {s : Set α} {a a₁ a₂ b b₁ b₂ c x : α}
+section Lattice
+variable [Lattice β] {f : α → β} {s : Set α} {a b : α}
+
+lemma _root_.MonotoneOn.mapsTo_uIcc (hf : MonotoneOn f (uIcc a b)) :
+  MapsTo f (uIcc a b) (uIcc (f a) (f b)) := by
+  rw [uIcc, uIcc, ←hf.map_sup, ←hf.map_inf] <;>
+    apply_rules [left_mem_uIcc, right_mem_uIcc, hf.mapsTo_Icc]
+#align monotone_on.maps_to_uIcc MonotoneOn.mapsTo_uIcc
+
+lemma _root_.AntitoneOn.mapsTo_uIcc (hf : AntitoneOn f (uIcc a b)) :
+  MapsTo f (uIcc a b) (uIcc (f a) (f b)) := by
+  rw [uIcc, uIcc, ←hf.map_sup, ←hf.map_inf] <;>
+    apply_rules [left_mem_uIcc, right_mem_uIcc, hf.mapsTo_Icc]
+#align antitone_on.maps_to_uIcc AntitoneOn.mapsTo_uIcc
+
+lemma _root_.Monotone.mapsTo_uIcc (hf : Monotone f) : MapsTo f (uIcc a b) (uIcc (f a) (f b)) :=
+  (hf.monotoneOn _).mapsTo_uIcc
+#align monotone.maps_to_uIcc Monotone.mapsTo_uIcc
+
+lemma _root_.Antitone.mapsTo_uIcc (hf : Antitone f) : MapsTo f (uIcc a b) (uIcc (f a) (f b)) :=
+  (hf.antitoneOn _).mapsTo_uIcc
+#align antitone.maps_to_uIcc Antitone.mapsTo_uIcc
```

Comment:
Did these lemmas exist in mathlib3? They don't appear in the porting dashboard, and if they're new they shouldn't be `#align`ed

Reply:
Indeed, copy-paste error